pkg/components/util/helm.go

```diff
@@ -201,12 +201,6 @@ func chartFromManifests(name string, manifests map[string]string) (*chart.Chart,
 				continue
 			}
 
-			// Drop Namespace resource as we take care of its creation at another level and we don't want resources to collide.
```

Comment:
I'd like to see this functionality `Allowing helm to create release namespace` should be a separate PR as this functionality affects how other components are installed not just `experimental-istio-operator`.

This way all other components can be properly sanitized to remove release namespace manifests from the charts.